implementation-details.md

```diff
@@ -414,20 +414,20 @@ Finally, all remaining `CrdsValue`s values are sent back to the origin of the pu
 
 ### Pull responses
 
-Pull responses contain a list of `CrdsValue`s filtered using the bloom filter provided in a pull request. These are the values that the node sending the pull request was missing in its `crds`.
+Pull responses contain a list of `CrdsValue`s filtered using the Bloom filter provided in a pull request. These are the values that the node sending the pull request was missing in its `crds`.
 
 When a node receives a pull response it processes its list of `CrdsValue`s:
```

Comment:
"its" here is vague:
> When a node receives a pull response it processes its list of `CrdsValue`s:

When a node receives a pull response, it processes the list of CrdsValues contained within the pull response: